siphon/catalog.py

```diff
@@ -581,7 +584,7 @@ def access_with_service(self, service):
                 provider = NC4Dataset
             except ImportError:
                 raise ImportError('OPENDAP access requires netCDF4-python to be installed.')
-        elif service == 'NetcdfSubset':
+        elif service in ('NetcdfSubset', 'NetcdfServer'):
```

Comment:
Would it be useful to make `('NetcdfSubset', 'NetcdfServer')` a variable (`ncssServiceNames`?), in case we need to do this special snowflake check elsewhere?

Reply:
Sure. I've updated the PR to do that and use it exclusively.